config/jobs/kubernetes-sigs/cloud-provider-azure/cloud-provider-azure-config.yaml

```diff
@@ -415,7 +415,7 @@ periodics:
   extra_refs:
     - org: kubernetes-sigs
       repo: cluster-api-provider-azure
-      base_ref: release-1.7
+      base_ref: release-1.8
```

Comment:
Successful test run after this change:

- https://prow.k8s.io/view/gs/kubernetes-jenkins/logs/cloud-provider-azure-master-capz/1634222734521143296